### PR TITLE
RHDEVDOCS-6088: Content creation for v1.12.4 RN

### PR DESCRIPTION
**Pull request by Dhruv-Soni11**
Version(s):

GitOps 1.12, GitOps 1.13

Issue:

https://issues.redhat.com/browse/RHDEVDOCS-6088

Link to docs preview:

[Release Notes for v1.12.4](https://78107--ocpdocs-pr.netlify.app/openshift-gitops/latest/release_notes/gitops-release-notes.html#gitops-release-notes-1-12-4_gitops-release-notes)

QE review:
- [ ] QE has approved this change.

SME review: @chetan-rns [anjoseph@redhat.com](mailto:anjoseph@redhat.com)
QE review: @varshab1210
Internal peer review: 
Peer review: @mramendi 

Additional information:

### CODE REVIEW

**Comment by varshab1210** [comment]
LGTM, thanks

**Comment by anandf** [comment]
LGTM

**Comment by chetan-rns** [comment]
LGTM

File: modules/gitops-release-notes-1-12-4.adoc
Comment (mramendi) [inline modules/gitops-release-notes-1-12-4.adoc:38]: I would suggest using monospace formatting for `passthrough` and `reencrypt` where they are mentioned, in order to highlight that these are settings.

I would change "who had configured" to "who previously configured" to help readers whose first language is not English - that is, if the "previously"/"had" part [meaning that the configuration was done before the change of the default policy] is actually important.  Otherwise just "users who configured".

If at all possible, I would also clarify exactly when the default policy changed (in a previous numbered version, presumably).

I would replace "Now," with "With this update," 

The Jira _is_ visible to external users here, that's good :)

**Review by mramendi — APPROVED** [review]
Looks great for peer review! A few comments - Jira access issues, style/language trickery, in one case a request for additional information if possible.

File: modules/gitops-release-notes-1-12-4.adoc
Comment (mramendi) [inline modules/gitops-release-notes-1-12-4.adoc:22]: make sure this becomes open to unauthorized users when you merge

File: modules/gitops-release-notes-1-12-4.adoc
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-12-4.adoc:24]: Yes, it is best kept as have and instead of if you installed to ensure consistency with previous releases.

File: modules/gitops-release-notes-1-12-4.adoc
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-12-4.adoc:34]: Hey @mramendi - The SME has advised us not to change the rights of the particular JIRAs so I am closing this comment now.

File: modules/gitops-release-notes-1-12-4.adoc
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-12-4.adoc:36]: https://github.com/openshift/openshift-docs/pull/78107#discussion_r1660484239

File: modules/gitops-release-notes-1-12-4.adoc
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-12-4.adoc:38]: Fixed.

File: modules/gitops-release-notes-1-12-4.adoc
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-12-4.adoc:22]: https://github.com/openshift/openshift-docs/pull/78107#discussion_r1660484239

**Comment by Dhruv-Soni11** [comment]
Hi @mramendi, I have fixed your comments and left my thoughts for JIRAs that non-authorized users cannot access. Could you please merge this PR if everything looks fine?

File: modules/gitops-release-notes-1-12-4.adoc
Comment (chetan-rns) [inline modules/gitops-release-notes-1-12-4.adoc:38]: @Dhruv-Soni11 @varshab1210 Since we have reverted the policy to Passthrough, should we add a line to inform the users to manually delete the `argocd-server-tls` secret? This is already present in the [upstream docs](https://argocd-operator.readthedocs.io/en/latest/usage/routes/#tls-reencrypt-mode), but we should make it more explicit. I'll confirm this once.

Comment (chetan-rns) [reply modules/gitops-release-notes-1-12-4.adoc:38]: Nope. We've agreed not to include it since users are not impacted. Please feel free to mark this as resolved.

**Comment by mburke5678** [comment]
@Dhruv-Soni11 I asked @kalexand-rh about the restriction on reporting internal Red Hat only bugs and she agreed that we cannot publish them in the release notes. I noticed that the two internal bugs are [not listed in the errata]([url](https://access.redhat.com/errata/RHSA-2024:4163)), likely for the same reason. Please remove them from the PR and we can merge. Thank you!

**Comment by Dhruv-Soni11** [comment]
> @Dhruv-Soni11 I asked @kalexand-rh about the restriction on reporting internal Red Hat only bugs and she agreed that we cannot publish them in the release notes. I noticed that the two internal bugs are not listed in the errata, likely for the same reason. Please remove them from the PR and we can merge. Thank you!

Hi @mburke5678 - Thanks for your comment. I agree with your reasoning that since these two issues are not visible to the unauthorized user, we must not include it in the release notes too. But before doing that, I would like to ensure that @anandf and @Srivaralakshmi agree regarding this. So, I am tagging them here and I would wait for them to reply before removing any content.

**Comment by anandf** [comment]
> > @Dhruv-Soni11 I asked @kalexand-rh about the restriction on reporting internal Red Hat only bugs and she agreed that we cannot publish them in the release notes. I noticed that the two internal bugs are not listed in the errata, likely for the same reason. Please remove them from the PR and we can merge. Thank you!
> 
> Hi @mburke5678 - Thanks for your comment. I agree with your reasoning that since these two issues are not visible to the unauthorized user, we must not include it in the release notes too. But before doing that, I would like to ensure that @anandf and @Srivaralakshmi agree regarding this. So, I am tagging them here and I would wait for them to reply before removing any content.

I am ok to remove the release note entries for the private JIRA tickets if that is the general practice.

**Comment by Srivaralakshmi** [comment]
> > @Dhruv-Soni11 I asked @kalexand-rh about the restriction on reporting internal Red Hat only bugs and she agreed that we cannot publish them in the release notes. I noticed that the two internal bugs are not listed in the errata, likely for the same reason. Please remove them from the PR and we can merge. Thank you!
> 
> Hi @mburke5678 - Thanks for your comment. I agree with your reasoning that since these two issues are not visible to the unauthorized user, we must not include it in the release notes too. But before doing that, I would like to ensure that @anandf and @Srivaralakshmi agree regarding this. So, I am tagging them here and I would wait for them to reply before removing any content.

@Dhruv-Soni11 Thanks for the updates, please remove the release note entries for the private JIRA tickets.

**Comment by Dhruv-Soni11** [comment]
Hi @mburke5678, I have removed the two issues from the content as you suggested after getting the required approvals. PTAL and let me know what you think. Thanks.

**Pull request merged**
